test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -59,9 +57,14 @@ var _ = Describe("Google Cloud Filestore CSI Driver", func() {
 		}()
 
 		// TODO validate the Filestore instance creation at the cloud provider layer
+		stageDir := filepath.Join("/tmp/", volName, "stage")
+		_, err = instance.SSH(fmt.Sprint("mkdir -p ", stageDir))
+
```

Comment:
Check the ssh error here?

Reply:
Actually, looking at the number of times we make directories and the cleanup we want to do after its done, maybe it would be better to at the beginning make a tmpdir (eg https://golang.org/pkg/io/ioutil/#TempDir), put all the staging & publishing directories in there, and then just remove the tmpdir at the end instead of removing all the intermediate directories each time.

Note the tmpdir might be necessary if tests are run in parallel on the same instance.